aio/aio-proxy/aio_proxy/search/helpers/helpers.py

```diff
@@ -2,7 +2,7 @@
 
 
 def extract_ul_and_etab_from_es_response(structure):
-    structure_dict = structure.to_dict(skip_empty=False, include_meta=False)
```

Comment:
Why did you remove `skip_empty=False` ?

Reply:
As explained in the description of the pull request, I had to remove those args to make it work.
Otherwise the app was raising this error : https://errors.data.gouv.fr/organizations/sentry/issues/132881/?project=14&query=is%3Aunresolved+is%3Afor_review+assigned_or_suggested%3A%5Bme%2C+none%5D&referrer=issue-stream&sort=inbox